# `python:3.6.5-alpine3.4`

## Docker Metadata

- Image ID: `sha256:5c527d46f6682cd1172faf07a38b522cc2e8e709274de2d07a7682dd76f8144f`
- Created: `2018-03-31T02:26:54.643244039Z`
- Virtual Size: ~ 90.42 Mb  
  (total size of all layers on-disk)
- Arch: `linux`/`amd64`
- Command: `["python3"]`
- Environment:
  - `PATH=/usr/local/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin`
  - `LANG=C.UTF-8`
  - `GPG_KEY=0D96DF4D4110E5C43FBFB17F2D347EA6AA65421D`
  - `PYTHON_VERSION=3.6.5`
  - `PYTHON_PIP_VERSION=9.0.3`
